client/state/ui/editor-deprecation-dialog/actions.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { EDITOR_DEPRECATION_DIALOG_IS_SHOWING } from 'state/action-types';
+
+export function showEditorDeprecationDialog() {
+	return {
+		type: EDITOR_DEPRECATION_DIALOG_IS_SHOWING,
+		isShowing: true,
+	};
+}
+
+export function hideEditorDeprecationDialog() {
```

Comment:
Why did you choose to use two different functions rather than one and a parameter?

Reply:
Copy/pasta from a different action, mostly. 🙂 I've deleted the one that isn't being used.